pkgs/development/libraries/libportal/default.nix

```diff
@@ -29,6 +29,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-wDDE43UC6FBgPYLS+WWExeheURCH/3fCKu5oJg7GM+A=";
   };
 
+  patches = [
+    # https://github.com/flatpak/libportal/pull/106
+    ./fix-build-on-darwin.patch
```

Comment:
Like I wrote earlier ideally we want to fetchpatch this. or is there something that prevents it?

Reply:
Just want to confirm that GitHub does cache the file even if I force push something.

Reply:
It has not been merged yet.

Reply:
> It has not been merged yet.


That's alright


> Just want to confirm that GitHub does cache the file even if I force push something.

but not if you clone master. There it is not included.

Reply:
Done.